templates/server-acl-init-job.yaml

```diff
@@ -89,6 +89,33 @@ spec:
                 {{- if not (or (and (ne (.Values.client.enabled | toString) "-") .Values.client.enabled) (and (eq (.Values.client.enabled | toString) "-") .Values.global.enabled)) }}
                 -create-client-token=false \
                 {{- end }}
+                {{- if .Values.global.enableConsulNamespaces }}
```

Comment:
```
                {{- if .Values.connectInject.enabled }}
                -create-inject-token=true \
                {{- end }}
```

Should switch to the new `-create-inject-auth-method` flag.

Reply:
It also needs the `-create-inject-namespace-token` flag, otherwise, the inject token is not created and the injector is permanently stuck in the init phase.

Reply:
Updated both. Put the inject namespace token flag in with the other namespace flags.